contrib/packages/debian/Makefile

```diff
@@ -0,0 +1,28 @@
+PROJECT=avocado
+
+all:
+	@echo
+	@echo "Debian related targets:"
+	@echo "deb:      Generate both source and binary debian packages"
+	@echo "deb-src:  Generate a source debian package"
+	@echo "deb-bin:  Generate a binary debian package"
+	@echo
+
+prepare-source:
+	# build the source package in the parent directory
+	# then rename it to project_version.orig.tar.gz
+	dch -D "vivid" -M -v "$(VERSION)" "Automated (make builddeb) build."
+	$(PYTHON) setup.py sdist $(COMPILE) --dist-dir=../ --prune
```

Comment:
This will not work as the location changed. But I can see that we have the same issue in `avocado`. Do you think it's worth fixing?

Reply:
But fixing it should be fairly simple so I'd really prefer that...

Reply:
Not happy about that, but I understand your point...